react-toggle-switch/my-app/src/Switch.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+type Props = {
+  SwitchFlip: string[];
+  SwitchCircle: string[];
+  onOff: string[];
+};
```

Comment:
Passing arrays here is a clever way to customize the Switch, so I can use it in multiple places with different looks. I like it!